forge-gui/res/cardsfolder/upcoming/ancient_bronze_dragon.txt

```diff
@@ -0,0 +1,10 @@
+Name:Ancient Bronze Dragon
+ManaCost:5 G G
+Types:Creature Elder Dragon
+PT:7/7
+K:Flying
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigRoll | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, roll a d20. When you do, put X +1/+1 counters on each of up to two target creatures, where X is the result.
+SVar:TrigRoll:DB$ RollDice | ResultSVar$ Result | Sides$ 20 | SubAbility$ DBPutCounter
```

Comment:
This one needs a reflexive trigger

Reply:
Thanks for the catch. Shifted reflexive trigger to the correct script